src/plugins/kibana/public/settings/sections/indices/_create.js

```diff
@@ -23,7 +23,7 @@ uiModules.get('apps/settings')
 
   // this and child scopes will write pattern vars here
   const index = $scope.index = {
-    name: 'logstash-*',
+    name: config.get('kibana.indexPatternPlaceholder', 'logstash-*'),
```

Comment:
Ok thanks for your feedback, I will try to find some time this week to finish off this improvement